kornia/sensor/camera/camera.py

```diff
@@ -0,0 +1,75 @@
+from __future__ import annotations
+
+from dataclasses import dataclass
+from enum import Enum
+from typing import Union
+
+from kornia.core import Tensor
+from kornia.geometry.vector import Vector2, Vector3
+from kornia.sensor.camera.distortion import AffineTransform, BrownConradyTransform
+from kornia.sensor.camera.projection import OrthographicProjection, Z1Projection
+
+
+class CameraDistortionType(Enum):
```

Comment:
Perhaps we can change this to `CameraModelType` and avoid the naming redundancy. It will then depart from the Sophus API though:
https://github.com/strasdat/Sophus/blob/9f1334964ad922cf5d9b771d1e87cd611cefc507/cpp/sophus/sensor/camera_model.h#L319

Reply:
That naming makes more sense. We don’t have strictly follow Sophus